client/reader/following-manage/index.jsx

```diff
@@ -115,6 +115,7 @@ class FollowingManage extends Component {
 							delaySearch={ true }
 							delayTimeout={ 500 }
 							placeholder={ searchPlaceholderText }
+							additionalClasses="following-manage__search-new"
```

Comment:
Nope, but there are two search boxes on the page: one for new subs, and one for existing subs, so I thought it was important to give them a different name.

Originally the styles for this input were targeted at `.following-manage .search`, which targets both search boxes.